apps/login/package.json

```diff
@@ -3,8 +3,19 @@
   "private": true,
   "scripts": {
     "dev": "next dev",
-    "test": "jest",
-    "test:watch": "jest --watch",
+    "test": "concurrently --group --timings --kill-others-on-fail 'npm:test:unit' 'npm:test:integration'",
+    "test:watch": "concurrently --kill-others 'npm:test:unit:watch' 'npm:test:integration:watch'",
+    "test:unit": "jest --config ./__test__/jest.config.ts",
+    "test:unit:watch": "pnpm test:unit --watch",
+    "test:integration": "pnpm mock:build && concurrently --group --names 'mock,test' --success command-test --kill-others 'pnpm:mock' 'ZITADEL_API_URL=http://localhost:22222 start-server-and-test dev http://localhost:3000 \"test:integration:run\"'",
+    "test:integration:watch": "concurrently --names 'mock,test' --kill-others 'pnpm:mock' 'ZITADEL_API_URL=http://localhost:22222 start-server-and-test dev http://localhost:3000 \"pnpm nodemon -e js,jsx,ts,tsx,css,scss --ignore \\\"__test__/**\\\" --exec \\\"pnpm test:integration:run\\\"\"'",
```

Comment:
We could watch changes directly with a watch command from cypress

Reply:
I would have liked to have https://github.com/zitadel/typescript/pull/28/commits/ae4c975a31c1b8a27caa1849241e7665ac203a22 working and I tried really hard. It would have improved the developer experience a lot. But I couldn't get the plugin https://github.com/TheBrainFamily/cypress-app-watcher-preprocessor to run.

`SyntaxError: 'import' and 'export' may appear only with 'sourceType: module'`

Do you know by chance how to fix that?

As I couldn't find any useful alternatives - that wait until client and server are ready - I reverted the commit.

Reply:
We probably need a `webpack.config.ts` for it to work 🤔

Reply:
I don't know if the cypress preprocessor is the best option and if we still need it once we require webpack anyways, so it is ok for me if we keep nodemon 👍

Reply:
It would be really nice if it worked, as with the preprocessor not the whole cypress process with the browser would restart but only the tests.

I'll quickly give it another try with a webpack config. But maybe I'll need some help with that.